### ISSUE TITLE
How to use EmbeddedRecordsMixin with EmberData 2.3.0?

### ISSUE THREAD

**Issue by nhemanth007** [emberjs/data#4123]
http://emberjs.com/blog/2016/01/12/ember-data-2-3-released.html

As mentioned in the above blog post, serializer `EmbeddedRecordsMixin` is not part of public API.

**Comment by pangratz** [comment]
This has been misssing and is addressed in #4125.

Until this is available in the next release, you can use `DS.EmbeddedRecordsMixin`:

```js
// app/serializers/application.js

import DS from "ember-data";
import RESTSerializer from "ember-data/serializers/rest";

export default RESTSerializer.extend(DS.EmbeddedRecordsMixin, {
  // ...
});
```

**Comment by pangratz** [comment]
Since #4125 has been merged, you can now import the `EmbeddedRecordsMixin` via modules:

```js
import RESTSerializer from "ember-data/serializers/rest";
import EmbeddedRecords from "ember-data/serializers/embedded-records-mixin";

export default RESTSerializer.exend(EmbeddedRecordsMixin, {
  // ...
});
```
